Feature/adding person client and employee classes

DianaMD21 opened the pull request:
This branch contains the Person, Client and Employee entities.

DianaMD21 committed:
Adding Person Entity

DianaMD21 committed:
Adding Client Entity

reynoldmorel commented on src/monegros-restaurant/entities/Person.java:
```diff
@@ -0,0 +1,60 @@
+        this.address = address;
+    }
+
+    public GregorianCalendar getBirthdate() {
```
No need of gregorian calendar here. Use `ZonedDateTime` maybe better.

reynoldmorel commented on src/monegros-restaurant/entities/Person.java:
```diff
@@ -0,0 +1,60 @@
+    public void setBirthdate(GregorianCalendar birthdate) {
+        this.birthdate = birthdate;
+    }
+}
```
Please add an enter here to avoid the warning

DianaMD21 committed:
Adding Employee Entity and adding Enter at the end of class

Changing GregorianCalendar variable to ZoneDateTime and adding Enter at the end of Class

Adding an Enter at the end of the class to avoid the warning

Revert "Adding an Enter at the end of the class to avoid the warning"

This reverts commit 9e41386118891727cd4c98fb0163250779abc41c.

Adding Enter at the end of the Class to avoid warning

Adding Enter at the end
